terra-batch-libchart/values.yaml

```diff
@@ -43,6 +43,9 @@ exports:
     config:
       resourceGroup: "UNDEFINED"
 
+    provenance:
+      sourceWorkspaceId: ""
```

Comment:
This needs to be blank or else when not a clone the value will be injected as the default "RUNTIME_PARAMETER"

Reply:
should it change to "" in coa-helm/values.yaml too?